cmd/gardenlet/app/app.go

```diff
@@ -460,6 +467,41 @@ func (g *garden) cleanupOrphanedExtensionsServiceAccounts(ctx context.Context, g
 	return flow.Parallel(taskFns...)(ctx)
 }
 
+// TODO(Kostov6): Remove this code after v1.91 has been released.
+func cleanupGRMSecretFinalizers(ctx context.Context, seedClient client.Client, log logr.Logger) error {
+	mrs := &resourcesv1alpha1.ManagedResourceList{}
+	if err := seedClient.List(ctx, mrs); err != nil {
+		if meta.IsNoMatchError(err) {
+			log.Info("Received a 'no match error' while trying to list managed resources. Will assume that the managed resources CRD is not yet installed (for example new Seed creation) and will skip cleaning up GRM finalizers.")
```

Comment:
```
Executing golangci-lint
cmd/gardenlet/app/app.go:475:13: structured logging message should not end with punctuation mark: "Received a 'no match error' while trying to list managed resources. Will assume that the managed resources CRD is not yet installed (for example new Seed creation) and will skip cleaning up GRM finalizers." (logcheck)
			log.Info("Received a 'no match error' while trying to list managed resources. Will assume that the managed resources CRD is not yet installed (for example new Seed creation) and will skip cleaning up GRM finalizers.")
```

Ups, I missed this when suggesting the change.

```suggestion
			log.Info("Received a 'no match error' while trying to list managed resources. Will assume that the managed resources CRD is not yet installed (for example new Seed creation) and will skip cleaning up GRM finalizers")
```